Changelog.md

```diff
@@ -1,27 +1,28 @@
-### 0.5.8 (unreleased)
+### 0.5.8 (2019-04-30)
 
 Important Bugfixes:
+ * Code Generator: Fix initialization routine of uninitialized internal function pointers in constructor context.
  * Yul Optimizer: Fix SSA transform for multi-assignments.
 
 
 Language Features:
- * Code Generation: Implement copying recursive structs from storage to memory.
  * ABIEncoderV2: Implement encoding of calldata arrays and structs.
+ * Code Generation: Implement copying recursive structs from storage to memory.
  * Yul: Disallow function definitions inside for-loop init blocks.
 
 
 Compiler Features:
  * ABI Decoder: Raise a runtime error on dirty inputs when using the experimental decoder.
- * Standartd JSON Interface: Metadata settings now re-produce the original 'useLiteralContent' setting from the compilation input.
+ * Standartd JSON Interface: Metadata settings now re-produce the original ``"useLiteralContent"`` setting from the compilation input.
+ * Optimizer: Add rule for shifts by constants larger than 255 for Constantinople.
+ * Optimizer: Add rule to simplify certain ANDs and SHL combinations
```

Comment:
Nit: I noticed that the bullets under Important bugfixes and Language features are sorted but these are not